docs/operations/set-up-auth-for-tensorzero.mdx

```diff
@@ -157,6 +157,12 @@ For example:
 docker compose run --rm gateway --create-api-key
 ```
 
+You can optionally set an expiration date and time for the new API key by providing the UTC timestamp as an argument, like so:
+
+```bash
+docker compose run --rm gateway --create-api-key "2025-12-20 23:00:00.000000 UTC"
```

Comment:
Could you make the argument an independent flag? We'll likely want to add additional arguments in the future (e.g. RBAC).

```
... --create-api-key --expiration-date "2025-12-20 23:00:00.000000 UTC"
```